### DISCUSSION TITLE
Is there any plan to make this software available for android or ios ?

### DISCUSSION THREAD

**Discussion by amiayan** [sumatrapdfreader/sumatrapdf#3976] [Q&A]
This pdf reader is lightweight , fast , supports opening multiple pdfs in a tab format . I wish we have the same experience on android tab or ipad .

**Comment by GitHubRulesOK** [comment]
AFAIK there is an Android version of MuPDF as I heard they provide the Android PDF code to Adobe?
but dont know what formats it supports

Ah here it is 
https://mupdf-appkit.readthedocs.io/en/latest/?utm_source=mupdf&utm_medium=website&utm_content=inline-link

>You are free to use the MuPDF App Kit for evaluation purposes. However, if you require to publish an application, then a commercial license is required. 

For more on how to use your license key see:

[Android License Key](https://mupdf-appkit.readthedocs.io/en/latest/android-getting-started.html#android-license-key)

[iOS License Key](https://mupdf-appkit.readthedocs.io/en/latest/ios-getting-started.html#ios-license-key)

Commercial license
A commercial license for MuPDF App Kit will be necessary before publishing your App project to an application store or other type of digital distribution platform for software applications. Please read the [Software License Terms and Conditions](https://artifex.com/appkit/terms/index.html) document in its entirety.

**Comment by kjk** [comment]
No. See https://www.sumatrapdfreader.org/docs/Why-only-Windows
